version: '3.8'
services:

  plex:
    image: plexinc/pms-docker
    ports:
      - 32400:32400
    volumes:
      - /home/docker/plex/plex_data:/config
      - /home/docker/plex/transcode:/transcode
      - /media/disks:/data
    environment:
      PLEX_CLAIM: 'claim-H1A4sGEANzmnpQjDkTk1'
      TZ: "America/Argentina/Buenos_Aires"
    restart: unless-stopped

  portainer:
    image: portainer/portainer-ce
    ports:
      - 9000:9000
    volumes:
      - /home/docker/portainer/portainer_data:/data
      - /var/run/docker.sock:/var/run/docker.sock
    restart: unless-stopped
    
  home-assistant:
    image: homeassistant/home-assistant
    ports:
      - 8123:8123
    volumes:
      - /home/docker/home_assistant/homeassistant_data:/config
    restart: unless-stopped

  tautulli:
    image: linuxserver/tautulli
    ports:
      - 8181:8181
    volumes:
      - /home/docker/tautulli:/config
    environment:
      TZ: "America/Argentina/Buenos_Aires"
      PUID: "1000"
      PGID: "1000"
    restart: unless-stopped


volumes:
  plex_data:
  plex_transcode:
  plex_media:
  portainer_data:
  homeassistant_data:
  tautulli: